src/StringComparator.java

```diff
@@ -0,0 +1,20 @@
+import org.junit.platform.commons.util.StringUtils;
+import java.util.Comparator;
+
+public class StringComparator implements Comparator<String> {
+
+    @Override
+    public int compare(String string1, String string2) {
+        if(string1 == string2) {
+            return 0;
+        }
+        if (StringUtils.isBlank(string1) || string1.length() < 2) {
+            return -1;
+        }
+        if (StringUtils.isBlank(string2) || string2.length() < 2) {
+            return 1;
+        }
+        return Character.compare(string1.charAt(1), string2.charAt(1));
```

Comment:
))) what the order for "tom" & "dog" will be? if the second letter is the same but after are different?

Reply:
Only one was added. Fixed it.